react-design/hannah-app/src/Banner.jsx

```diff
@@ -0,0 +1,3 @@
+export default function Banner({ items }) {
```

Comment:
You're really only displaying a single item, so maybe it should be `item` instead of `items`.

Reply:
Got it!